packages/components/inputs/selectable-search-input/src/selectable-select.tsx

```diff
@@ -0,0 +1,195 @@
+import { useIntl } from 'react-intl';
+import Select, {
+  components,
+  type SingleValueProps,
+  type Props as ReactSelectProps,
+} from 'react-select';
+import {
+  createSelectStyles,
+  DropdownIndicator,
+  messages,
+} from '@commercetools-uikit/select-utils';
+import { designTokens, useTheme } from '@commercetools-uikit/design-system';
+import { type ReactNode, useCallback } from 'react';
+import type {
+  TSelectableSearchInputProps,
+  TOption,
+} from './selectable-search-input';
+
+type TBase = {
+  backgroundColor?: string;
+  color?: string;
+};
+
+type TCreateSelectableSelectStyles = {
+  isDisabled?: boolean;
+  hasError?: boolean;
+  hasWarning?: boolean;
+  isReadOnly?: boolean;
+  menuPortalZIndex?: number;
+  isNewTheme: boolean;
+  dropdownHasFocus?: boolean;
+  textInputHasFocus?: boolean;
+};
+
+type TSingleValue = {
+  id?: string;
+  children?: ReactNode;
+} & SingleValueProps;
+
+const SingleValue = ({ id, ...props }: TSingleValue) => (
+  <components.SingleValue {...props}>
+    <label htmlFor={id}>{props.children}</label>
+  </components.SingleValue>
+);
+
+const createSelectableSelectStyles = ({
+  hasWarning,
+  hasError,
+  isDisabled,
+  isReadOnly,
+  menuPortalZIndex,
+  isNewTheme,
+  dropdownHasFocus,
+  textInputHasFocus,
+}: TCreateSelectableSelectStyles) => {
+  const selectStyles = createSelectStyles({
+    hasWarning,
+    hasError,
+    menuPortalZIndex,
+    isNewTheme,
+  });
+
+  return {
+    ...selectStyles,
+    control: (base: TBase, state: ReactSelectProps) => ({
+      ...selectStyles.control(base, state),
+      padding: designTokens.paddingForSelectableSearchInputDropdown,
```

Comment:
**nit**: Since we already have a styles file for the component, perhaps we can move this to it.

Reply:
Updated [cd72889](https://github.com/commercetools/ui-kit/pull/2426/commits/cd72889b1c645bc9d296d204c3808bf5f9450647)